codex/contracts/market.nim

```diff
@@ -306,6 +306,17 @@ method subscribeSlotFreed*(market: OnChainMarket,
     let subscription = await market.contract.subscribe(SlotFreed, onEvent)
     return OnChainMarketSubscription(eventSubscription: subscription)
 
+method subscribeSlotReservationsFull*(
+  market: OnChainMarket,
+  callback: OnSlotReservationsFull): Future[MarketSubscription] {.async.} =
+
+  proc onEvent(event: SlotReservationsFull) {.upraises:[].} =
```

Comment:
Not relevant for this PR, but we should probably get rid of `upraises` and replace it with regular `raises` at some point.